packages/frontend/src/pages/Token/Metrics.tsx

```diff
@@ -144,7 +143,7 @@ export default function TokenMetrics() {
           <Text.HeadlineSmall color="text2">${memecoin.symbol}</Text.HeadlineSmall>
         </Row>
 
-        <Box position="relative">
+        <Box position="relative" ref={dropdownRef}>
```

Comment:
It would be even better with a Dropdown component in `/src/components/Dropdown/`.
With usage such as:

```jsx
<Dropdown>
  {/* dropdown content */}
</Dropdown>
```

Reply:
Makes sense 👍🏼

Reply:
Updated. Please let me know what are your thoughts on the new approach.